README.adoc

```diff
@@ -1,122 +1,353 @@
+If it illustrates your product, it probably goes here.
+If it brands your company, it probably goes in `theme/assets/`.
+
+content/pages/:::
+For AsciiDoc files of the _page_ content type.
+
+content/snippets/:::
+For content _snippets_.
+
+content/topics/:::
+For AsciiDoc files of the core _topic_ content type.
+
+data/::
+All YAML small-data files that contain content-relevant information go here.
+These data files differ from those that belong in `_configs/` (or `_ops/`) in important ways, essentially revolving around whether the data needs to be available for display.
+If it is not establishing settings or used to inform non-build functions (like in `_ops/`), the data file probably belongs in `data/`.
+Let's look at some key data files standard to LDCMF.
+
+data/meta.yml:::
+For general information about your company, URL and path info.
+This file usually contains just simple data: a big (or small) column of basic key-value pairs to create simple variables.
+
+data/products.yrml:::
+For subdivided information about your products in distinct blocks.
+Each block can be called for selective ingest during build routines using the colon signifier, such as by calling `data/products.yml:product-1`, where `product-1:` is a top-level block in the `products.yml` file.
+
+data/guides.yml:::
+This block is for content-oriented data that is distinct between the different portals or guides you're producing.
+This is often redundant to your `products.yml` file, if product editions themselves are the major point of divergence in your docs, and it is formatted the same way.
+For _this project_ (LDCMF Guides), the _guides_ are oriented toward _audiences_ (documentarians, admins, and developers), but the products (LiquiDoc and LDCMF) are distinct from this and actually documented/instructed _together_ in each guide.
++
+[TIP]
+Favoring the filename `products.yml` is conventional when products and guides (portals) have a 1:1 relationship and `guides.yml` file is superfluous.
+
+data/schema.yml:::
+Can also be `data/manifest.yml`, this crucial file provides, central manifest of all page-forming content items (pages, topics) and how they are organized (metadata such as categories into which content items fall).
+The schema file carries essential build info that lets us see relationships between topics and build content-exclusive portals from otherwise fairly dumb, decontextualized repositories.
+
+data/terms.yml:::
+By no means a required file, `terms.yml` is a great example of a file that is really just for content.
+You can have as many of these key-value files, serving whatever purposes you may wish.
+
+products/::
+This is an optional path for LDCMF projects.
+If you plan to embed your product repos as submodules, the `products/` directory is the base path to stick them in.
+For LDCMF Guides, this path effectively leads to symlinks for the LiquiDoc and LDCMF repos, so any files within those repos are accessible to be drawn into our docs.
```

Comment:
Is the symlink thing also true, in theory, for other projects that use submodules? If so, let's generalize that a bit more.